File: FinalProject/WebApplication10/WebApplication10/Models/Request.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;
using WebApplication10.BL;

namespace WebApplication10.Models
{
    public class Request
    {

        Request()
        {

            status = RequestStatus.NotTaked;
        }
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int RequestID { get; set; }
        public string Location { get; set; }
        public string Destination { get; set; }
        public DateTime RequestTime { get; set; }
        public DateTime TripTime { get; set; }
        public string ShippingType {get;set;}
        public string carType { get; set; }
   
        public RequestStatus status { get; set; }

        [ForeignKey("Client")]
        public string ClientID { get; set; }
        public Client Client { get; set; }

    }
}

File: FinalAdmin/src/app/ViewModel/request.ts
export class Request {
    constructor
    (
        public Location:string,
        public Destination:string,
        public RequestTime:Date,
        public TripTime:Date,
        public ShippingType:string,
        public carType:string,
        public  ClientName:string,
        public  DriverName :string,
        public  Bill: DoubleRange,
        public  clientRate :number, 
        public  DriverRate :number
    )
    {

    }
}
